frontend/src/app/app.component.html

```diff
@@ -2,13 +2,14 @@
   <div class="row">
     <div class="col">
       <div >
-        <app-filters-form></app-filters-form>
+        <app-tasks-filters></app-tasks-filters>
+        <button  class="btn btn-outline-secondary" (click)="showTaskForm()">Add task</button>
```

Comment:
You have two spaces after `<button` and before `class=`. There should be only one space.

Reply:
Done